bin/pycbc_hdf5_splitbank

```diff
@@ -114,6 +106,10 @@ if args.mchirp_sort:
 
 # Split the templates in the bank taken as input into the smaller banks
 
+# If an array of filenames
+if args.output_filenames:
+    args.number_of_banks = len(args.output_filenames)
+
```

Comment:
This is a repetation of lines 86-88. So it needs to be removed in either of the two places.